include/mbedtls/psa_util.h

```diff
@@ -94,6 +94,7 @@ static inline psa_algorithm_t mbedtls_psa_translate_cipher_mode(
                 return( PSA_ALG_CBC_NO_PADDING );
             /* Intentional fallthrough for taglen != 0 */
             /* fallthrough */
+            MBEDTLS_FALLTHROUGH;
```

Comment:
Any envisioned issues with the stray semicolon that results when this is defined as empty?

Reply:
I can define it with the semicolon in if its thought better. But people are still likely to stick a semi-colon on the end. This shouldn't cause any issues, though.

Reply:
Classically, compilers required a comment containing the word “fallthrough”. I think we need to keep the comment. We should keep it after the attribute, because I don't know if compilers would skip an empty macro.